datastore/migrate/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM migrate/migrate:4
+
+COPY data/migrations /migrations
+
+COPY ./run-migrate.sh /
+
+RUN chmod +x /run-migrate.sh
```

Comment:
This line is not needed, as `run-migrate.sh` is marked `+x` in the repo. Looks like the `COPY` keeps that.

Reply:
Ah, yes. Will fix it, originally it was not, but the pre-commit complained when a file with a shebang was not executable.